usaspending_api/common/management/commands/check_for_endpoint_documentation.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from usaspending_api.common.helpers.endpoint_documentation import (
+    CURRENT_ENDPOINT_PREFIXES, get_endpoint_urls_doc_paths_and_docstrings, get_endpoints_from_endpoints_markdown,
+    validate_docs
+)
+
+
+class Command(BaseCommand):
+
+    help = "Checks endpoints to ensure they point to documentation and are contained in the master endpoint list."
+
+    def handle(self, *args, **options):
+        master_endpoint_list = get_endpoints_from_endpoints_markdown()
+
+        endpoints = get_endpoint_urls_doc_paths_and_docstrings(CURRENT_ENDPOINT_PREFIXES)
+
+        messages = []
+        for endpoint in endpoints:
+            messages.extend(validate_docs(*endpoint, master_endpoint_list))
+
```

Comment:
Minor request: Can you add a check to count or list the endpoints which have a document however the doc isn't an API contract? This shouldn't make the check not pass. But it would be useful to track the v2 endpoints without a contract.